Tests/PNDeviceIndependentMatcher.m

```diff
@@ -41,7 +41,7 @@ - (BOOL)_compareComponents:(NSURLComponents *)components toOtherComponents:(NSUR
     if (![components.scheme isEqualToString:otherComponents.scheme]) {
         return NO;
     }
-    if (![components.host isEqualToString:otherComponents.host]) {
+    if (![components.host isEqualToString:@"msgfiltering-dev.pubnub.com"]) {
```

Comment:
@jzucker2 Is it possible to make matcher not sensitive to host? I believe we shouldn't expose test addresses.